spec/integration/users_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
```

Comment:
- [optional] Great work with the test. it would have been better to put all seperate test in different spec files. therefore they were suppose to be files for...
   - User index page
   - User show page
   - User post index page
   - User post show page